exercises/chat/online_examples/pub_server.py

```diff
@@ -1,22 +1,23 @@
-#http://learning-0mq-with-pyzmq.readthedocs.org/en/latest/pyzmq/patterns/pubsub.html
+# http://learning-0mq-with-pyzmq.readthedocs.org/en/latest/pyzmq/patterns/pubsub.html
 
-import zmq
 import random
 import sys
 import time
 
+import zmq
+
 port = "5556"
 if len(sys.argv) > 1:
-    port =  sys.argv[1]
+    port = sys.argv[1]
     int(port)
 
 context = zmq.Context()
 socket = context.socket(zmq.PUB)
 socket.bind("tcp://*:%s" % port)
 
 while True:
-    topic = random.randrange(9999,10005)
-    messagedata = random.randrange(1,215) - 80
-    print "%d %d" % (topic, messagedata)
-    socket.send("%d %d" % (topic, messagedata))
-    time.sleep(1)
\ No newline at end of file
+    topic = random.randrange(9999, 10005)
+    messagedata = random.randrange(1, 215) - 80
+    print("%d %d" % (topic, messagedata))
+    socket.send_string("%d %d" % (topic, messagedata))
```

Comment:
```suggestion
    socket.send_string("{} {}".format(topic, messagedata))
```
… I think you get the point :wink: